# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/util-linux/hotfix-bsd-disklabel.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- util-linux-2.38.1/libfdisk/src/bsd.c	2022-05-11 13:13:10.877531766 +0500
+++ util-linux-2.38.1-fixed/libfdisk/src/bsd.c	2023-07-18 20:56:54.209681000 +0500
@@ -378,7 +378,7 @@
 	int rc, yes = 0;
 	struct bsd_disklabel *d = self_disklabel(cxt);
 
-	fdisk_info(cxt, _("The device %s does not contain BSD disklabel."), cxt->dev_path);
+	fdisk_info(cxt, _("The device %s does not contain BSD disklabel."), cxt->parent->dev_path);
 	rc = fdisk_ask_yesno(cxt,
 			_("Do you want to create a BSD disklabel?"),
 			&yes);
